resources/sdk/clisdkclient/scripts/combineApiDataFiles.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
```

Comment:
Can this script be moved to a shared location instead of duplicated in each folder? This kind of duplication is very likely to result in the scripts getting out of sync at some point in the future.